packages/joint-vitest-plugin-mock-svg/package.json

```
@@ -63,8 +63,8 @@
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
     "@vitejs/plugin-react": "^4.3.4",
-    "@vitejs/plugin-vue": "^4.2.3",
-    "@vue/eslint-config-typescript": "^11.0.3",
+    "@vitejs/plugin-vue": "^5.2.4",
+    "@vue/eslint-config-typescript": "14.5.0",
```

```suggestion
    "@vue/eslint-config-typescript": "^14.5.0",
```

in this case, we should use `^` version